schemas/com.snowplowanalytics.iglu/igluctl_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,201 @@
+					"description": "Redshift database schema (e.g. 'atomic')",
+					"type": "string",
+            		"minLength": 1,
+            		"maxLength": 128,
+					"default": "atomic"
+				},
+				"owner": {
+					"description": "Redshift table owner",
+					"type": "string",
+					"maxLength": 128
+				},
+				"db": {
+					"description": "DB to which we need to generate DDL",
+					"type": "string",
+					"minLength": 1,
+					"maxLength": 64,
+					"default": "redshift"
+				},
+				"varcharSize": {
+					"description": "Redshift varchar size",
+					"type": "integer",
+					"minimum": 1,
+					"maximum": 65535,
+					"default": 4096
+				},
+				"withJsonPaths": {
+					"description": "Produce JSON Paths files with DDL or not",
+					"type": "boolean",
+					"default": true
+				},
+				"rawMode": {
+					"description": "Produce raw DDL with Snowplow-specific data or not",
+					"type": "boolean",
+					"default": false
+				},
+				"splitProduct": {
+					"description": "Split product types (like [string,integer]) into separate columns or not",
+					"type": "boolean",
+					"default": false
+				},
+				"noHeader": {
+					"description": "Place header comments into output DDL or not",
+					"type": "boolean",
+					"default": false
+				},
+				"force": {
+					"description": "Force overwrite existing manually-edited files",
+					"type": "boolean",
+					"default": false
+				}
+			}
+		},
+		"actions": {
+			"description": "Schema for s3cp and push commands",
+			"type": ["array", "null"],
+			"items": {
+				"type": "object",
+				"anyOf":[
+					{
+						"description": "Schema for push command",
+						"type": "object",
+						"properties": {
+							"action": {
+								"type": "string",
+								"enum": ["push"]
+							},
+							"registry": {
+								"type": "string",
+								"format": "uri"
+							},
+							"isPublic": {
+								"type": "boolean",
+								"default": false
+							},
+							"apikey": {
+								"oneOf":[
+									{ "format": "uuid" },
+									{ "pattern": "^\\$[A-Z0-9-_]+" }
+								]
+							}
+						}
+					},
+					{
+						"description": "Schema for s3cp command",
+						"type": "object",
+						"properties": {
+							"action": {
+								"type": "string",
+								"enum": ["s3cp"]
+							},
+							"uploadFormat": {
+								"type": "string",
+								"enum": [
+									"jsonpath",
+									"jsonschema"
+								]
+							},
+							"bucketPath": {
+								"type": "string",
+								"pattern": "^s3://"
+							},
+							"accessKeyId": {
+								"description": "AWS Access Key Id",
+								"type": "string",
+								"pattern": "^[A-Z0-9-_]+"
+							},
+							"secretAccessKey": {
+								"description": "AWS Secret Access Key",
+								"type": "string",
+								"pattern": "^[A-Z0-9-_]+"
+							},
+							"profile": {
+								"description": "AWS profile",
+								"type": "string"
+							},
+							"region": {
+								"description": "AWS region",
+								"type": "string",
+								"enum": [
```

Comment:
I wouldn't do that. AWS SDK is able to upload files to region that appeared after this particular release of AWS SDK. And user will be blocked by this schema if new regions appears.